<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSender</class>
 <widget class="QWidget" name="FormSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Отправка</string>
  </property>
  <widget class="QTextEdit" name="textMailEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Текст письма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendTo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Кому отправить?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Добавить файлы (&lt;путь к файлу&gt;&amp;&lt;путь к файлу&gt;...)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
